CMakeLists.txt

```diff
@@ -147,8 +152,7 @@ endif()
 set(CMAKE_PREFIX_PATH ${NUPIC_CORE})
 set(CMAKE_INCLUDE_PATH "${NUPIC_CORE}/include")
 
-add_definitions(-DNTA_PLATFORM_${PLATFORM}${BITNESS}
-                -DHAVE_CONFIG_H
+add_definitions(-DHAVE_CONFIG_H
```

Comment:
maybe the `-DNA_ARCH_32/64` can be set here as `-DNTA_ARCH_${BITNESS}`? but that's just a matter of taste..